src/blog/2022/12/flowforge-joins-openjs-foundation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: FlowForge joins the OpenJS Foundation
```

Comment:
Do we join them? Or support them?

Reply:
We have joined as members.

Reply:
Could change to FlowForge Inc. joins...?

Reply:
I think that's better indeed.

```suggestion
title: FlowForge Inc. joins the OpenJS Foundation
```

Reply:
> Ok, it sounds like we've donated FlowForge. We donated, I think it's different.

Changing to: 'FlowForge Inc becomes a member of the OpenJS Foundation' to remove any confusion